### DISCUSSION TITLE
gatsby build fails with error: No "exports" main defined in @babel/helper-compilation-targets/package.json

### DISCUSSION THREAD

**Discussion by adamjw3** [vercel/vercel#5645] [General] [answered]
have a gatsby build hosted and its started erroring

11:54:12.978 | success onPostBootstrap - 0.186s
-- | --
11:54:12.980 | ⠀
11:54:12.982 | info bootstrap finished - 4.523 s
11:54:12.983 | ⠀
11:54:13.514 | failed Building production JavaScript and CSS bundles - 0.494s
11:54:13.519 | ERROR #98123  WEBPACK
11:54:13.519 | Generating JavaScript bundles failed
11:54:13.519 | [BABEL] /vercel/workpath0/.cache/production-app.js: No "exports" main defined in /vercel/workpath0/node_modules/@babel/helper-compilation-targets/package.json
11:54:13.519 | File: .cache/production-app.js
11:54:13.551 | not finished run queries - 0.553s
11:54:13.627 | npm ERR! code ELIFECYCLE
11:54:13.627 | npm ERR! errno 1
11:54:13.631 | npm ERR! pup@0.1.0 build: `gatsby build`
11:54:13.631 | npm ERR! Exit status 1
11:54:13.631 | npm ERR!
11:54:13.631 | npm ERR! Failed at the pup@0.1.0 build script.
11:54:13.631 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
11:54:13.642 | npm ERR! A complete log of this run can be found in:
11:54:13.642 | npm ERR!     /vercel/.npm/_logs/2021-01-06T11_54_13_632Z-debug.log
11:54:13.648 | Error: Command "npm run build" exited with 1
11:54:15.860 | Done with "package.json"

Anyone now how to fix this

**Comment by styfle** [comment, marked as answer]
The answer is in this babel issue: https://github.com/babel/babel/issues/11216

> If you hit this issue from search engines, please update all `@babel` deps to v7.8.7 or above.

If that dependency is coming from gatsby or a plugin, you'll probably need to update that too.
